### DISCUSSION TITLE
bluesky account @ web.brid.gy

### DISCUSSION THREAD

**Discussion by lucajet** [snarfed/bridgy-fed#1035] [General]
Sometime, searching accounts from the fediverse, i see accounts like

@whatever.bsky.social@web.brid.gy

which, if i guess it correctly, are accounts bridged by "bridgy" instead of "bridgy fed".

It's adding confusion

**Comment by snarfed** [comment]
These aren't a Bridgy classic vs Fed thing, these are web sites bridged into the fediverse by Bridgy Fed. That's what it originally launched in 2017 to do; the Bluesky part is the newcomer. 😁

Details in the docs, eg https://fed.brid.gy/docs#web-enhanced . I'm open to ideas if you think they're missing anything or could be better advertised somehow! Otherwise, tentatively closing.

**Comment by snarfed** [comment]
...oh wait, you mean _Bluesky_ accounts on web.brid.gy? Agreed, that is confusing. That will happen if someone enters their Bluesky handle into the "web site" form on https://fed.brid.gy/, but you're right, we shouldn't allow it. I'll fix that.

**Comment by lucajet** [comment]
exactly this, maybe someone created those on classic bridgy *before* you came out with bridgy fed .

**Reply by snarfed** [reply]
They're still Bridgy Fed, Bridgy classic doesn't make its own fediverse accounts at all, but I can definitely block this in Bridgy Fed going forward.
